### ISSUE TITLE
Link to the Knative Slack from the Knative Twitter Account

### ISSUE THREAD

**Issue by omerbensaadon** [knative/community#706]
![Screen Shot 2021-07-23 at 4 38 31 PM](https://user-images.githubusercontent.com/16281246/126838993-03da877a-0e1d-467c-885b-ecb6e6c1faaa.png)

Crossplane does this and I think it would be a good idea. It is our community's preferred way of reaching us. 

* Expected benefits. Who gains the benefits? Why will they benefit?
More folks joining the Slack, asking questions
* Expected costs. Who bears the costs? How heavy are they?
None 
* Timeframe for implementation / rollout.
Basically instantly
* Are you willing to drive the process, or is this a request for help?
I can't I don't have the permissions

**Comment by vaikas** [comment]
I tried to see if I could this from the tweetdeck but couldn't figure out how to do it. I think somebody needs to login to the twitter account and do it from there, but I do not know how.

**Comment by vaikas** [comment]
@thisisnotapril is this something that you might be able to help with? I can't do this with tweetdeck so we'd need the "real" twitter account to actually login as knative.
